Thanks.  Will re-test this on the same Ubuntu 16.04.6 VM today.

I left VM running (with build from original changes) with the `kolibri manage importcontent...` command since yesterday.  This morning I got this error log: I don't know if this is relevant so I'm just posting it here:

```bash
user@ubuntu:~$ kolibri manage importcontent network 1ceff53605e55bef987d88e0908658c5
...
... I lost the results here due to the scrollback limit of the Terminal.
...
  [########################################################---------------------------------------------]   56%  10:40:30WARNING  increment argument should be an integer for progressbars to work
  [###############################################################################################################]  100%WARNING  increment argument should be an integer for progressbars to work
WARNING  total argument should be an integer for progressbars to work
WARNING  increment argument should be an integer for progressbars to work
WARNING  increment argument should be an integer for progressbars to work
WARNING  increment argument should be an integer for progressbars to work
...
... warnings repeated 5x
...
ERROR    An error occurred during content import: 520 Server Error:  for url: https://studio.learningequality.org/content/storage/6/1/61497dc0f7d71c1e23bdd9e7237d5baf.vtt
INFO     Setting availability to True of 58228 LocalFile objects based on passed in checksums
INFO     Setting availability of non-topic ContentNode objects based on LocalFile availability in 9 batches of 10000
INFO     Annotating ContentNode objects with children for 5 levels
INFO     Annotating ContentNode objects with children for level 5
INFO     Annotating ContentNode objects with children for level 4
INFO     Annotating ContentNode objects with children for level 3
INFO     Annotating ContentNode objects with children for level 2
INFO     Annotating ContentNode objects with children for level 1
WARNING  increment argument should be an integer for progressbars to work
  [#########################################################--------------------------------------------]   57%  10:24:19Traceback (most recent call last):
  File "/usr/bin/kolibri", line 11, in <module>
    load_entry_point('kolibri==0.13.2b1.dev0+git.47.g47aaec40', 'console_scripts', 'kolibri')()
  File "/usr/lib/python3/dist-packages/kolibri/dist/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/kolibri/dist/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/kolibri/dist/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/kolibri/utils/cli.py", line 243, in invoke
    return super(KolibriDjangoCommand, self).invoke(ctx)
  File "/usr/lib/python3/dist-packages/kolibri/dist/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/kolibri/dist/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/kolibri/dist/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/lib/python3/dist-packages/kolibri/utils/cli.py", line 641, in manage
    execute_from_command_line(["kolibri manage"] + ctx.args)
  File "/usr/lib/python3/dist-packages/kolibri/dist/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/lib/python3/dist-packages/kolibri/dist/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/lib/python3/dist-packages/kolibri/dist/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/lib/python3/dist-packages/kolibri/dist/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/usr/lib/python3/dist-packages/kolibri/core/tasks/management/commands/base.py", line 118, in handle
    return self.handle_async(*args, **options)
  File "/usr/lib/python3/dist-packages/kolibri/core/content/management/commands/importcontent.py", line 439, in handle_async
    renderable_only=options["renderable_only"],
  File "/usr/lib/python3/dist-packages/kolibri/core/content/management/commands/importcontent.py", line 159, in download_content
    renderable_only=renderable_only,
  File "/usr/lib/python3/dist-packages/kolibri/core/content/management/commands/importcontent.py", line 353, in _transfer
    raise exception
  File "/usr/lib/python3/dist-packages/kolibri/core/content/management/commands/importcontent.py", line 305, in _transfer
    f, filetransfer, overall_progress_update
  File "/usr/lib/python3/dist-packages/kolibri/core/content/management/commands/importcontent.py", line 409, in _start_file_transfer
    retry = retry_import(e, skip_404=True)
  File "/usr/lib/python3/dist-packages/kolibri/core/content/utils/import_export_content.py", line 159, in retry_import
    raise e
  File "/usr/lib/python3/dist-packages/kolibri/core/content/management/commands/importcontent.py", line 374, in _start_file_transfer
    with filetransfer, self.start_progress(
  File "/usr/lib/python3/dist-packages/kolibri/core/content/utils/transfer.py", line 102, in __enter__
    self.start()
  File "/usr/lib/python3/dist-packages/kolibri/core/content/utils/transfer.py", line 162, in start
    self.response.raise_for_status()
  File "/usr/lib/python3/dist-packages/kolibri/dist/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 520 Server Error:  for url: https://studio.learningequality.org/content/storage/6/1/61497dc0f7d71c1e23bdd9e7237d5baf.vtt
user@ubuntu:~$ 
```

This could be the [520 Web Server Returned an Unknown Error](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#520) as per Wikipedia.  Here's another reference I found: [Community Tip - Fixing Error 520: Web server is returning an unknown error](https://community.cloudflare.com/t/community-tip-fixing-error-520-web-server-is-returning-an-unknown-error/44205).  A note, I was also downloading updates from Steam in a Windows machine and that stopped also when I checked this morning.

I didn't look at the code, but maybe we can add this error code in the exception handler to prevent crashes like this?  An alternative perhaps is a retry mechanism of some sort?

--- a/kolibri/core/tasks/management/commands/base.py
+++ b/kolibri/core/tasks/management/commands/base.py
@@ -2,7 +2,6 @@ import abc
 import logging
 import sys
 from collections import namedtuple
-from numbers import Integral
 
 import click
 from django.core.management.base import BaseCommand
@@ -30,9 +29,6 @@ class ProgressTracker:
         self.level = level
         self.update_callback = update_callback
 
-        # Check that the total is an integer, otherwise progress bars will be unpredictable
-        if type(total) is not Integral:
-            logger.warn("total argument should be an integer for progressbars to work")
         # Also check that we are not running Python 2:
         # https://github.com/learningequality/kolibri/issues/6597
         if sys.version_info[0] == 2:
@@ -42,6 +38,8 @@ class ProgressTracker:
             # as we only want to display progress bars from the command line.
             try:
                 click.get_current_context()
+                # Coerce to an integer for safety, as click uses Python `range` on this
+                # value, which requires an integer argument
                 # N.B. because we are only doing this in Python3, safe to just use int,
                 # as long is Py2 only
                 self.progressbar = click.progressbar(length=int(total), width=0)

